vmfloaty.gemspec

```diff
@@ -6,10 +6,10 @@ require 'vmfloaty/version'
 Gem::Specification.new do |s|
   s.name = 'vmfloaty'
   s.version = Vmfloaty::VERSION
-  s.authors = ['Brian Cain']
-  s.email = ['brianccain@gmail.com']
+  s.authors = ['Puppet']
```

Comment:
I mentioned this in the other PR, but please do not erase me as the author of this tool. I was ok with transferring ownership, but erasing my name from this as the sole person who spent a lot of time writing and supporting this tool is not something I'm ok with.

Reply:
Agreed, we should continue to credit Brian here.

Reply:
Update incoming to #69

Reply:
fixed over in that pr